docs/events-challenges/community-events.md

```diff
@@ -0,0 +1,47 @@
+# Community Events
+
+Community events are a great way to connect with other players, showcase your creativity, and enjoy everything Theatria has to offer as a community. Events often start in Discord, and everyone is encouraged to participate or contribute their ideas.
+
+## How to Get Involved
+
+### Suggest an Event
+- **Join our Discord**: Start by joining Theatria's Discord server if you haven't already. This is where event planning begins.
+- **Share your idea**: Use the designated **#event-suggestions** channel to propose your event. Provide details such as:
+  - Event name
+  - Goals or activities
+  - Date and time (or a range of possible dates)
+  - Any resources or permissions needed
+
+### Join an Event
+- **Check the schedule**: Look for upcoming events announced in the **#events** channel on Discord.
```

Comment:
This gives me an idea for somehow having a shared calendar so people know of any upcoming events etc. For now though maybe we should rename the bullet point since there's no real schedule to speak of.

Maybe something like "Check for announcements"